qa/rpc-tests/mergetoaddress_helper.py

```diff
@@ -282,16 +281,10 @@ def run_test(self, test):
             test.nodes[1].generate(1)
         test.sync_all()
 
```

Comment:
The change to this part of the test is implemented by the parameter changes in `mergetoaddress_sprout`. The comment here is still accurate as it is referring to Sapling activation.

Reply:
Nit:  It's surprising that `mergetoaddress_sprout.py` and `mergetoaddress_sapling.py` both import `MergeToAddressHelper` but `mergetoaddress_mixednote.py` does not.
I believe this is part of a wider set of patterns that should be addressed _after_ the RPC tests are ported to Python3 in #4074 .

Reply:
> The change to this part of the test is implemented by the parameter changes in `mergetoaddress_sprout`. The comment here is still accurate as it is referring to Sapling activation.

I believe that above means, that the test behaves correctly in the `sprout` regime because the parameters of the `MergeToAddressHelper` consructor have been changed, and different arguments are passed for

Sprout: `'sprout', 'ANY_SPROUT', 800, 800, 0`
VS.
 Sapling: `'sapling', 'ANY_SAPLING', 800, 800, 0`

I don't think the PR should be blocked on test reorganization, but If I Understand Correctly (IIUC) flow logic specific to different test regimes (e.g. Sprout vs. Sapling) ought to be moved out of framework (e.g. "helper) code.   That kind of upgrade should wait until the RPC tests have been ported to Python3 in #4074 .